workflows/rnaseq/main.nf

```diff
@@ -216,21 +218,32 @@ workflow RNASEQ {
         //
         if (params.with_umi) {
             // Deduplicate genome BAM file before downstream analysis
-            BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME (
-                ch_genome_bam.join(ch_genome_bam_index, by: [0]),
-                params.umitools_dedup_stats
-            )
-            ch_genome_bam       = BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.bam
-            ch_genome_bam_index = BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.bai
-            ch_multiqc_files = ch_multiqc_files.mix(BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.deduplog.collect{it[1]})
-            ch_multiqc_files = ch_multiqc_files.mix(BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.stats.collect{it[1]})
-            ch_multiqc_files = ch_multiqc_files.mix(BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.flagstat.collect{it[1]})
-            ch_multiqc_files = ch_multiqc_files.mix(BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS_GENOME.out.idxstats.collect{it[1]})
+            if (params.umi_dedup_tool == "umicollapse") {
+                BAM_DEDUP_STATS_SAMTOOLS_UMICOLLAPSE_GENOME (
+                    ch_genome_bam.join(ch_genome_bam_index, by: [0])
+                )
+                umi_dedup_genome = BAM_DEDUP_STATS_SAMTOOLS_UMICOLLAPSE_GENOME
```

Comment:
```suggestion
                UMI_DEDUP_GENOME = BAM_DEDUP_STATS_SAMTOOLS_UMICOLLAPSE_GENOME
```
Capitalise so I don't get confused later and think this is a channel or something